modules/api_loader.lua

```diff
@@ -51,8 +64,7 @@ local function LoadParamsInFunction(param, interface)
   return name, data
 end
 
-
-
+--- Load Enums values
```

Comment:
@LuxoftAKutsan,
No it's a LDoc comments.
It's better to read HOWTO first before asking such questions ;)